### PR TITLE
Contact Form Milestone  

### PR DESCRIPTION
**Pull request by obed300**
In this milestone, I added a contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,196 +12,41 @@
...
-                    <p class="skills-one">Skills</p>
-                    <i class='fa fa-angle-right'></i>
-                </div>
-                <hr>
-            </section>
-        </div>    
+            </div>
+            <form action="https://formspree.io/f/{form_id}" method="post">
+                <ul>
+                    <li>
```
Comment (roshan-bajgain) [inline index.html:29]: - [ ] You should collect the data by using the [Formspree service](https://formspree.io/html). please create an account and then use the id that is generated not the `form_id` 😃

File: index.html [hunk 2]
```diff
@@ -12,196 +12,41 @@
...
+                        <input type="email" id="mail" name="user_email" placeholder="natajayanj14@gmail.com" required />
+                    </li>
+                    <li>
+                        <textarea id="msg" name="user_message" placeholder="Write your message here" maxlength="500" required ></textarea>
+                    </li>
+                    <li class="button">
+                        <button id="button" type="submit">Get It Touch</button>
+                      </li>
+                </ul>
+            </form>
```
Comment (roshan-bajgain) [inline index.html:42]: - [ ] You should Implement the button interactions (enable, hover, etc.). Please see the image below for reference
### Figma design
sssss

File: index.html [hunk 3]
```diff
@@ -12,196 +12,41 @@
...
+                    </li>
+                    <li>
+                        <input type="email" id="mail" name="user_email" placeholder="natajayanj14@gmail.com" required />
+                    </li>
+                    <li>
+                        <textarea id="msg" name="user_message" placeholder="Write your message here" maxlength="500" required ></textarea>
+                    </li>
+                    <li class="button">
+                        <button id="button" type="submit">Get It Touch</button>
+                      </li>
```
Comment (roshan-bajgain) [inline index.html:40]: - [ ] I would suggest you please stick as much as possible to the design. please remove the black border and provide padding to the placeholder
### Your design
Screenshot 2022-10-05 183440
### Fimga design
figa

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @obed300, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
- [ ] You have implemented almost everything for the contact form but you removed the code that you did last week and the previous day. please implement all things that need to be done. See the image below and please do it.
### only design on whole page
your
### Things that need to implement.
figmaaaaaa

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Qoosim) [reply index.html]: > * [ ]  I would suggest you please stick as much as possible to the design. please remove the black border and provide padding to the placeholder
> 
> ### Your design
> Screenshot 2022-10-05 183440
> 
> ### Fimga design
> figa

- [ ] This required changes has not been implemented. Kindly give it attention and make it look like the guiding figma template

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

-  No linter errors. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 




 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mckent05 — APPROVED** [review]
Hi @obed300,

# STATUS: APPROVED ✔️ ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well-done! 🎉 

## Highlights
- Your repo looks professional 👏 
- Descriptive commit messages 👍 👍 

## Optional suggestions
- Please I'd suggest that you make sure your site is responsive. This will ensure that your portfolio fits properly across all screen sizes 🙏 🙏 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**